CHANGELOG/CHANGELOG-3.2.md

```diff
@@ -1,3 +1,11 @@
+# Release notes for v3.2.1
```

Comment:
We update the changelog in the release branch. There is already a 3.2.1

Reply:
It should be 3.2.2 then?

Reply:
The change log in https://github.com/kubernetes-csi/external-provisioner/blob/v3.2.0/CHANGELOG/CHANGELOG-3.2.md and https://github.com/kubernetes-csi/external-provisioner/blob/v3.2.1/CHANGELOG/CHANGELOG-3.2.md
 is identical?

Reply:
hmm looks like we missed updating the changelog for v3.2.1 then.